app/src/main/kotlin/codes/chrishorner/socketweather/home/HomeForecastPresenter.kt

```diff
@@ -27,6 +33,16 @@ class HomeForecastPresenter(view: View) {
     currentIcon.setImageDrawable(context.getWeatherIconFor(forecast.iconDescriptor, forecast.night))
     currentTemp.text = forecast.currentTemp.formatAsDegrees(context)
     feelsLikeTemp.text = forecast.tempFeelsLike?.formatAsDegrees(context) ?: "--"
+    humidity.text = forecast.humidity?.formatAsPercent(context) ?: "--"
+    wind.text = forecast.wind.run { "$speed_kilometre km/h $direction" }
```

Comment:
This should use a localisable string. Maybe consider adding something like this to `strings.xml`

```xml
<string name="home_wind">%1$d km/h %2$s</string>
```